dashboard/app/models/script.rb

```diff
@@ -333,8 +333,10 @@ def self.lesson_extras_script_ids
     @@lesson_extras_script_ids ||= all_scripts.select(&:lesson_extras_available?).pluck(:id)
   end
 
-  def self.maker_units
-    @@maker_units ||= visible_units.select(&:is_maker_unit?)
+  def self.maker_units(user)
+    total_units = visible_units
+    total_units += all_scripts.select {|s| s.has_pilot_access?(user)}
+    @@maker_units ||= total_units.select(&:is_maker_unit?)
```

Comment:
I think there's a bug here where the first user's pilot units will be cached for all subsequent users.

Reply:
Thanks for caching this (ha)! Fixed now.